scripts/setup/flush-memcached

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh -xe
 
 # Flush memcached
-echo 'flush_all' | nc localhost 11211
+echo 'flush_all\nquit' | nc localhost 11211
```

Comment:
Will take a look at that, for me the initial version didn't work. I think it's a netcat thing actually, as it doesn't quit after executing the command. Unfortunately we have the `-w` switch on OpenBSD to set a timeout, while on Linux it's `-q`. It does fix the problem too.